### PR TITLE
Clarifies some parts of the assessment sheets

### PR DESCRIPTION
**Pull request by huangs1**
Edit some parts of the assessment sheets to clarify points.

### CODE REVIEW

File: assessment.md [hunk 1]
```diff
@@ -244,17 +244,19 @@ when creating python programs.
...
 * Managing software complexity
   * N = No complexity management techniques utilized
-  * I = Code is non-obvious and complex
-  * A = Basic code management is utilized, such as code simplification or encapsulation
-  * G = Most code is well managed using code complexity management techniques
-  * E = All code utilizes many complexity reduction techniques, such as modular design
+  * I = Code is frequently non-obvious and complex
+  * A = Basic code management is used, such as code simplification or encapsulation
+  * G = Most code is managed using code complexity management techniques such as
+    continous process
```
Comment (Michionlion) [inline assessment.md]: Typo: `continous` -> `continuous`

File: assessment.md [hunk 2]
```diff
@@ -244,17 +244,19 @@ when creating python programs.
...
+  * E = All code effecitively utilizes many complexity reduction techniques such
```
Comment (Michionlion) [inline assessment.md]: Typo: `effecitively` -> `effectively`

**Review by Michionlion — CHANGES_REQUESTED** [review]

File: assessment.md (hunk 2 above)
Comment (huangs1) [reply assessment.md]: I have fixed the typo.

File: assessment.md (hunk 1 above)
Comment (huangs1) [reply assessment.md]: I have fixed the typo.

**Review by Michionlion — APPROVED** [review]

**Review by Lancasterwu — APPROVED** [review]
This looks good

**Review by sutterj — APPROVED** [review]

**Review by toccinAC — APPROVED** [review]

**Pull request merged**